src/components/Figure.js

```diff
@@ -2,20 +2,31 @@ import React from 'react';
 import styled from '@emotion/styled';
 import CaptionLegend from './CaptionLegend';
 import Image from './Image';
+import { getImageAlignmentStyle } from '../utils/get-image-alignment-style';
 import { getNestedValue } from '../utils/get-nested-value';
 import { size } from './dev-hub/theme';
 
 const Figure = styled('figure')`
     margin: 0 0 ${size.articleContent};
+    width: ${({ figWidth }) => figWidth};
+    ${({ alignStyle }) => alignStyle};
 `;
-export default ({ nodeData, ...rest }) => (
-    <Figure
-        className="figure"
```

Comment:
I don't think we are using this `className` but could be wrong

Reply:
Yup, it's used [in docs-tools](https://github.com/mongodb/docs-tools/search?l=CSS&q=figure) but y'all don't depend on that!